apps/accounts/urls.py

```diff
@@ -151,4 +154,24 @@
         TemplateView.as_view(template_name="provider_portal/before_starting.html"),
         name="before-starting",
     ),
+    path(
+        "domain-hash/",
+        DomainHashHomeView.as_view(),
+        name="provider-hash-index",
+    ),
+    path(
+        "domain-hash/new/",
+        DomainHashCreateView.as_view(),
+        name="provider-hash-create",
+    ),
+    path(
+        "domain-hash/detail/<int:pk>/",
```

Comment:
and `/provider/<int:pk>/verified-domains/<str:domain>`

I had used the `id` before because I couldn't rely on them being unique to a specific provider, but with a single provider we might already have the disambiguation we need:

`/domain-hash/detail/int:pk`